features/notifications/build.gradle

```
@@ -5,8 +5,15 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 apply from: rootProject.file('gradle/common.gradle')
+android {
+    dataBinding.enabled = true
```

忘れないでね

---

まじこれはどっちでもいいけど
```suggestion
    android.dataBinding.enabled = true
```
ともかけるよい

---

あー，これしからないんだったらいちいちブロック作る必要ないからこれがいいね 👍